README.md

```diff
@@ -10,6 +11,8 @@ $$ T(n) =
     \end{cases}
 $$
 
+T(n)= $\Theta$ (log n)
```

Comment:
Never use = as it is incorrect. Please use the correct notation we've used in class.